tests/functional/index-non-auth.js

```diff
@@ -30,7 +30,7 @@ define([
     'reporter addon link is shown': function() {
       return this.remote
         .get(require.toUrl(url('/')))
-        .findByCssSelector('.wc-Navbar-link').getVisibleText()
+        .findByCssSelector('.js-Navbar-link').getVisibleText()
```

Comment:
Do we want to have a consistent style for `js-` classes/ids? Some of them are all lowercase, some mixed-case. Maybe it doesn't really matter...